blueprints/component-addon/index.js

```diff
@@ -50,7 +50,14 @@ module.exports = {
     var pathName       = [addonName, 'components', fileName].join('/');
 
     if (options.pod) {
-      pathName = [addonName, 'components', fileName,'component'].join('/');
+      var podModulePrefix = options.project.config().podModulePrefix || '';
+      var podPath = podModulePrefix.substr(podModulePrefix.lastIndexOf('/') + 1);
+      if (podPath) {
+        pathName = [addonName, podPath, 'components', fileName,'component'].join('/');
```

Comment:
Previously, if a podModulePrefix was used the path from the app component to the addon component was missing the prefix and would break.